config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -47,6 +47,57 @@ presubmits:
           requests:
             cpu: 7
             memory: 12Gi
+  - name: pull-kubernetes-verify-go-canary
+    cluster: k8s-infra-prow-build
+    always_run: false
+    skip_report: true
+    decorate: true
+    skip_branches:
+    - release-\d+.\d+ # per-release job
+    annotations:
+      fork-per-release: "true"
+      testgrid-num-failures-to-alert: '10'
+      testgrid-alert-stale-results-hours: '24'
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: sig-release-releng-informing, sig-testing-canaries
+    path_alias: k8s.io/kubernetes
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      # TODO(releng): Remove if we are able to support verify tests using the releng-ci image
+      #- image: gcr.io/k8s-testimages/kubekins-e2e:v20210226-c001921-go-canary
+      - image: k8s.gcr.io/releng/releng-ci:v0.4.0
```

Comment:
we really don't need more of these image environments to keep track of. we already have variants for this?